modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -150,11 +150,30 @@ export default function UpperToolbar({displayNames, isView, languageIds}) {
 		let alertMessage;
 
 		if (!definitionTitle) {
-			alertMessage = Liferay.Language.get('name-workflow-before-publish');
-
+			alertMessage = Liferay.Language.get(
+				'name-workflow-before-publish'
+			);
+			setAlertMessage(alertMessage);
+			setAlertType('danger');
+			setShowAlert(true);
```

Comment:
Hi @M-IgorFranca what do you think about create a method to make this code lines reusable?

Something like:
```
function _setAlert(alertMessage, alertType, showAlert) {
        setAlertMessage(alertMessage);
	setAlertType(alertType);
	setShowAlert(showAlert);
}
```

If it's possible, we can use this method in everywhere we need it.